mdbenchmark/analyze.py

```diff
@@ -169,17 +167,13 @@ def analyze(directory, plot, ncores):
         # with equal settings
         uniqueness = df.apply(lambda x: x.nunique())
         if uniqueness['gromacs'] > 1 or uniqueness['host'] > 1:
-            click.echo(
-                '{} Cannot plot benchmarks for more than one GROMACS module'
-                ' and/or host.'.format(
-                    click.style('ERROR', fg='red', bold=True)))
-            sys.exit(1)
+            console.error(
+                'Cannot plot benchmarks for more than one GROMACS module'
+                ' and/or host.')
```

Comment:
It is common practice to have trailing whitespaces on multiline strings. This is true across programming languages.